### PR TITLE
arrays

### PR DESCRIPTION
**Pull request by dariamagiera**
zmiany zmiany

### CODE REVIEW

File: rooms/wc.cpp [hunk 1]
```diff
@@ -56,38 +57,19 @@ Room wc(__attribute__((unused)) GameData& gameData) {
 
   if (yes()) {
```
Comment (seqre) [inline rooms/wc.cpp]: It doesn't have any sense. You demand player to say yes/no to a statement. Maybe change text into question?

File: rooms/wc.cpp [hunk 2]
```diff
@@ -1,8 +1,8 @@
 #include <iostream>
-
+#include <string>
 #include "../main.hpp"
 
-Room wc(__attribute__((unused)) GameData& gameData) {
+Room wc(__attribute__((unused)) GameData &gameData) {
```
Comment (phoe) [inline rooms/wc.cpp]: This should actually be `GameData& gamedata` - the change was unnecessary.

**Review by phoe — CHANGES_REQUESTED** [review]

File: rooms/wc.cpp (hunk 2 above)
Comment (apardyl) [reply rooms/wc.cpp]:
```suggestion
Room wc(__attribute__((unused)) GameData& gameData) {
```

Comment (SzymonSmykala) [reply rooms/wc.cpp]: @dariamagiera ping

**Review by SzymonSmykala — APPROVED** [review]

**Review by phoe — APPROVED** [review]

**Pull request merged**